linkerd/proxy/http/src/client.rs

```diff
@@ -126,11 +137,8 @@ where
                 wants_h1_upgrade: _,
                 was_absolute_form,
             } => {
-                let exec =
-                    tokio::executor::DefaultExecutor::current().instrument(info_span!("http1"));
-
+                #[allow(deprecated)]
```

Comment:
Out of curiosity, is there something else we should be doing instead? Or should we open a discussion on hyper about how to satisfy our use case?

Reply:
The `Builder::keep_alive` function was deprecated. After another look at the docs, it turns out it's actually pretty easy to use the new API (you set `pool_max_idle_per_host` to 0 to disable keepalive), so I've removed this in f21809a430b440b3e49d87197c6891e4b36bf04a